src/sdk-reference/cpp/1/offline-tools/index.md

```diff
@@ -7,69 +7,74 @@ order: 100
 
-## queueMaxSize
+### queue_max_size
 
-A writable `number` defining the maximun size of the `offlineQueue`.
+An `unsigned long` defining the maximun size of the `offline_queue`.
 
-## queueTTL
+Default value: `500`
 
-A writable `number` defining the time in milliseconds a queued request is kept in the `offlineQueue`.
+### queue_ttl
 
-## startQueuing()
+An `unsigned` defining the time in milliseconds a queued request is kept in the `offline_queue`.
 
-Starts the requests queuing. Request will be put in the `offlineQueue` instead of being discarded, until `stopQueuing` is called.
-Works only in `offline` state, and if the `autoQueue` option is set to false. Call `playQueue` to send to Kuzzle the
-requests in the queue, once the SDK state passes to `online`. Call `flushQueue` to empty the queue without sendint the requests.
+Default value: `120000`
 
-## stopQueuing()
+### auto_queue
 
-Stop queuing the requests. Requests will no more be put in the `offlineQueue`, they will be discarded.
-Works only in the `offline` state, and if the `autoQueue` option is set to `false`.
+A `boolean` telling the SDK whether to automatically queue requests during the `offline` state or not.
 
-## playQueue()
+Default value: `false`
 
-Sends to Kuzzle all the requests in the `offlineQueue`. Works only if the SDK is not in a `offline` state, and if the 
-`autoReplay` option is set to false.
+### auto_replay
 
-## flushQueue()
+A `boolean` telling the SDK whether to automatically send or not the requests in the `offline_queue` on a
+`reconnected` event.
 
-Empties the `offlineQueue` without sending the requests to Kuzzle.
+Default value: `false`
 
-## autoQueue
+### auto_reconnect
 
-A writable `boolean` telling the SDK whether to automatically queue requests during the `offline` state or not.
+A `boolean` telling the SDK whether to automatically reconnect or not to Kuzzle after a connection loss.
 
-## autoReplay
+Default value: `true`
 
-A writable `boolean` telling the SDK whether to automatically send or not the requests in the `offlineQueue` on a
-`reconnected` event.
+### reconnection_delay 	
+
+A `unsigned long` specifying the time in milliseconds between different reconnection attempts.
+
+Default value: `10000`
+
+### auto_resubscribe
+
+A `boolean` telling the SDK whether to automatically renew or not all subscriptions on a reconnected event.
 
-## autoReconnect
+Default value: `true`
 
-A writable `boolean` telling the SDK whether to automatically reconnect or not to Kuzzle after a connection loss.
+## Methods
 
-## reconnectionDelay 	
+### [startQueuing()]({{ site_base_path }}sdk-reference/cpp/1/kuzzle/start-queuing/)
 
-A read-only `number` specifying the time in milliseconds between different reconnection attempts.
+Starts queuing requests when in `offline` state. Request will be put in the `offline_queue` instead of being discarded, until `stopQueuing` is called.
```

Comment:
```suggestion
Starts queuing requests when in `offline` state. Requests will be put in the `offline_queue` array instead of being discarded, until `stopQueuing` is called.
```